service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerTest.java

```diff
@@ -80,4 +90,29 @@ void shouldReturnUnsuccessfulResponseWithMalformedData() throws Exception {
                 .content("Mock"))
             .andExpect(status().is4xxClientError());
     }
+
+    @Test
+    void shouldCallNotify() throws Exception {
+        Map<String, String> expectedParameters = ImmutableMap.<String, String>builder()
+            .put("court", "")
+            .put("localAuthority", "")
+            .put("orders", "")
+            .put("directionsAndInterim", "")
+            .put("timeFrame", "")
+            .put("reference", CASE_ID)
+            .put("caseUrl", "")
+            .build();
+
+        mockMvc
+            .perform(post("/callback/case-submission")
+                .header("authorization", AUTH_TOKEN)
+                .header("user-id", USER_ID)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(readBytes("fixtures/case.json")))
```

Comment:
We should use `core-case-data-store-api/populated-case-details.json` here and assert that parameters map that we pass to GOV.UK Notify is exactly what we expect.